javascript-data-model/main.js

```diff
@@ -0,0 +1,24 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    order1Header: [{ orderNo: '114-3941689-8772232' }, { datePlaced: 'August 4, 2020' }, { total: 34 }, { shipTo: 'JS Masher' }, { delivered: '08/08/2020' }],
+    order1Item1: [{ type: 'book', title: 'JavaScript for impatient programmers', author: 'Dr. Axel Rauschmayer', returnWindow: '09/07/2020', price: 31.55 }]
```

Comment:
Same thing as above, this property name can be made to be more general, such as `orderedItems`